src/nethsec/mwan/__init__.py

```diff
@@ -156,7 +157,10 @@ def store_rule(e_uci: EUci, name: str, policy: str, protocol: str = None,
     e_uci.set('mwan3', rule_config_name, 'rule')
     e_uci.set('mwan3', rule_config_name, 'label', name)
     e_uci.set('mwan3', rule_config_name, 'use_policy', policy)
-    e_uci.set('mwan3', rule_config_name, 'sticky', '0')
+    # test if sticky is True or False, if not raise an error
+    if type(sticky) is not bool:
+        raise ValidationError('sticky', 'sticky_not_valid', sticky)
+    e_uci.set('mwan3', rule_config_name, 'sticky', '1' if sticky else '0')
```

Comment:
You [_should_](https://pypi.org/project/pyuci/#:~:text=bool%3A%20this%20is,sets%20this%20type.) be able to leave just `sticky`
```suggestion
    e_uci.set('mwan3', rule_config_name, 'sticky', sticky)
```